packages/core/src/components/AutocompleteField/AutocompleteField.scss

```diff
@@ -0,0 +1,117 @@
+// Form-specific variables can be found in _variables.forms
+@import '@cmsgov/design-system-support/src/settings/index';
+
+/*
+Autocomplete field
+
+@paypal https://github.com/paypal/downshift#itemtostring
```

Comment:
This isn't going to render anything in the docs since it's not a supported "flag". We use `@uswds` for USWDS links since we do want to render those on the page.

Reply:
Done. Removed from docs.